src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -264,11 +265,9 @@ protected virtual void Initialize()
                     TraceWriter = new ConsoleTraceWriter(hostTraceLevel);
                 }
 
-                _debugModeFileWatcher = new FileSystemWatcher(hostLogPath, ScriptConstants.DebugSentinelFileName)
-                {
-                    EnableRaisingEvents = true
-                };
-                _debugModeFileWatcher.Created += OnDebugModeFileChanged;
```

Comment:
Hmm, the code that is checked in also subscribes to Changed, but I don't see that line removed here? Strange.